app/Http/Controllers/Service/Admin/TradersController.php

```diff
@@ -172,16 +189,46 @@ public function update(AdminNewUpdateTraderRequest $request, int $id)
                 // users should be an array; sync them to our trader
                 $t->users()->sync($userIds);
 
+                // did we disable a trader up there?
+                if (array_key_exists('disabled_at', $fillArray) &&
+                    !is_null($fillArray['disabled_at'])
+                ) {
+                    // force log-out any users who are or were in a newly *disabled* trader
+                    $affectedUserIds = array_unique(array_merge($origUserIds, $userIds));
+
+                    User::with(['tokens'])
+                        ->whereIn('id', $affectedUserIds)
+                        ->each(function ($user) {
+                            // knicked from the loginProxy
+                            $accessToken = $user->token();
+                            if ($accessToken) {
+                                Log::info('removing refresh tokens'. $accessToken);
```

Comment:
Would it be better to log user id? Is this ever multiple tokens? or should be 'removing access token' ?